runtime/shared_common/OSCachesysv.cpp

```diff
@@ -2021,6 +2021,11 @@ SH_OSCachesysv::getJavacoreData(J9JavaVM *vm, J9SharedClassJavacoreDataDescripto
 	}
 #endif
 	descriptor->cacheDir = _cachePathName;
+#if defined(WIN32)
```

Comment:
Please add the missing comments (`/* !defined(WIN32) */`) on lines 2008, 2013 and 2022 above.